### PR TITLE
Graphics2D example

### PR DESCRIPTION
**Pull request by dima-avdeev-jb**
https://youtrack.jetbrains.com/issue/COMPOSE-357/Create-new-multi-example-project-2d-graphics

 - Merged non UI examples visual-effects, falling-balls and minesweeper into one **graphics2d**
 - Removed deprecated function calls `kotlin.system.getTimeNanos()`
 - Little bit simplifyed code
 - Used Material3

image

### CODE REVIEW

File: examples/graphics-2d/shared/src/commonMain/kotlin/Graphics2D.kt [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                            Button(onClick = {
+                                exampleState.value = it
+                            }) {
+                                Text(it.name)
+                            }
+                        }
+                    }
+                } else {
+                    example.content { w, h ->
+                        requestWindowSize(w, h + TOP_APP_BAR_HEIGHT)
```
Comment (pjBooms) [inline examples/graphics-2d/shared/src/commonMain/kotlin/Graphics2D.kt:77]: May we add some padding or center choses?

File: examples/graphics-2d/androidApp/build.gradle.kts [hunk 2]
```diff
@@ -17,9 +17,9 @@ kotlin {
 
 android {
     compileSdk = 34
-    namespace = "org.jetbrains.fallingballs"
+    namespace = "org.jetbrains.Graphics2D"
```
Comment (pjBooms) [inline examples/graphics-2d/androidApp/build.gradle.kts:20]: wrong case, should be org.jetbrains.graphics2d -- else Android version does not start

**Review by pjBooms — CHANGES_REQUESTED** [review]

**Review by pjBooms — APPROVED** [review]

File: examples/validateExamplesIos.sh [hunk 3]
```diff
@@ -33,9 +33,7 @@ runGradle() {
 
 runGradle chat
 runGradle codeviewer
-runGradle falling-balls
 runGradle imageviewer
 runGradle todoapp-lite
-runGradle visual-effects
 runGradle widgets-gallery
-runGradle minesweeper
+runGradle graphics-2d
```
Comment (dima-avdeev-jb) [inline examples/validateExamplesIos.sh:39]: @igordmn
Changed in scripts as well

File: examples/README.md [hunk 4]
```diff
@@ -4,9 +4,7 @@
 | [Imageviewer](imageviewer) | Image Viewer application                                                                           | Android, iOS, Desktop |
 | [Codeviewer](codeviewer) | File browser and code viewer application                                                           | Android, iOS, Desktop |
 | [Chat](chat) | A simple chat                                                                                      | Android, iOS, Desktop |
-| [Minesweeper](minesweeper) | A simple game where you need to find hidden mines                                                  | Android, iOS, Desktop |
-| [Falling Balls](falling-balls) | A simple game                                                                                      | Android, iOS, Desktop |
-| [Visual effects](visual-effects) | Visual effects                                                                                     | Android, iOS, Desktop |
+| [Graphics2D](graphics-2d)            | 2D Games and graphics examples                           | Android, iOS, Desktop |
```
Comment (dima-avdeev-jb) [inline examples/README.md:7]: @igordmn 
changed links as well

File: examples/graphics-2d/README.md [hunk 5]
```diff
@@ -16,6 +16,10 @@ Choose a run configuration for an appropriate target in Android Studio and run i
 
 `./gradlew desktopApp:run`
 
-## Run native on MacOS
-Choose **shared[macosX64]** or **shared[macosArm64]** configuration in IDE and run it.
+## Run experimental JS on Browser
```
Comment (igordmn) [inline examples/graphics-2d/README.md:19]: Let's remove this section. We don't write public docs for the Web JS target. The official web target uses wasm, and the JS target is an experimental implementation and a road to the wasm web implementation

File: examples/graphics-2d/shared/build.gradle.kts [hunk 6]
```diff
@@ -48,11 +48,11 @@ kotlin {
 
     val enableKjsWorkaround = project.properties["workaround.kotlin.js.kt60852"] == "true"
 
-    fun KotlinDependencyHandler.addCommonDependencies() {
+    fun  KotlinDependencyHandler.addCommonDependencies() {
```
Comment (igordmn) [inline examples/graphics-2d/shared/build.gradle.kts:51]:
```suggestion
    fun KotlinDependencyHandler.addCommonDependencies() {
```

File: examples/graphics-2d/shared/build.gradle.kts [hunk 7]
```diff
@@ -15,7 +15,7 @@ kotlin {
     androidTarget()
 
     jvm("desktop")
-
+
```
Comment (igordmn) [inline examples/graphics-2d/shared/build.gradle.kts:18]:
```suggestion

```

File: examples/graphics-2d/README.md (hunk 5 above)
Comment (pjBooms) [reply examples/graphics-2d/README.md:19]: We may replace JS to WASM, when we merge WASM to master. I would leave JS target

Comment (dima-avdeev-jb) [reply examples/graphics-2d/README.md:19]: I also think it's better to leave this section in the README. This is an experimental target, but it works. 
But, I suggest to chage title. Let's call it simple: **Run experimental on Browser** (without JS mentions)

@igordmn What do you think?

**Review by igordmn — APPROVED** [review]

File: examples/graphics-2d/README.md (hunk 5 above)
Comment (dima-avdeev-jb) [reply examples/graphics-2d/README.md]: @igordmn Done, removed

**Pull request merged**